unstructured/utils.py

```diff
@@ -552,11 +551,9 @@ def identify_overlapping_case(
             overlapping_case = "partial overlap with duplicate text"
 
         else:
-            (
-                largest_ngram_percentage,
-                _,
-                largest_n,
-            ) = calculate_largest_ngram_percentage(text1, text2)
+            (largest_ngram_percentage, _, largest_n) = calculate_largest_ngram_percentage(
+                text1, text2
+            )
```

Comment:
Okay, yeah, no, my mistake. This doesn't fit on one line even without the left-hand-side (LHS) parens. The LHS parens should go though. Black will still wrap but won't add them back in.